### PR TITLE
Set the master account as the signer for regular key signed transaction

### PR DESCRIPTION
**Pull request by sgkim126**
This PR fixes the bug that made the signer of the regular key signed transaction does not exist.

It can have a potential performance issue. I cannot run the test with more than 1000 transactions in a block. However, it can be the problem of jest. When I ran it with `yarn start`, the performance is not so bad.

### CODE REVIEW

**Comment by sgkim126** [comment]
I made Travis skip the tests that send many transactions to the chain.
Instead of it, I added the tests that call `workerpool.getSigners` directly.

**Comment by sgkim126** [comment]
Remove ts-jest doesn't make tests fast enough.
The test time becomes 192 seconds(https://travis-ci.com/sgkim126/codechain-indexer/builds/109265249) from 210 seconds(https://travis-ci.com/sgkim126/codechain-indexer/builds/109263477). But it's not fast enough to test more than 100 transactions.

**Comment by sgkim126** [comment]
@joojis I made the set-regular-key test use mocha.
It's much faster than jest.

**Comment by sgkim126** [comment]
I'll rebase it on https://github.com/CodeChain-io/codechain-indexer/pull/250

**Comment by sgkim126** [comment]
I'll re-request the review after https://github.com/CodeChain-io/codechain-indexer/pull/251

**Comment by sgkim126** [comment]
PR updated

File: src/models/logic/utils/workerpool.ts [hunk 1]
```diff
@@ -4,56 +4,116 @@ import {
...
+            txs.map(async tx => {
+                const pubKey = tx.getSignerPublic().value;
+                const regularKeyOwner = await getRegularKeyOwnerByPublicKey(
+                    pubKey,
+                    { transaction: options.transaction }
+                );
+                if (regularKeyOwner != null) {
+                    return regularKeyOwner;
+                }
+                return PlatformAddress.fromPublic(tx.getSignerPublic().value, {
```
Comment (joojis) [inline src/models/logic/utils/workerpool.ts]: Would you change it to reuse `pubkey`? `getSingerPublic` isn't cheap.

Comment (sgkim126) [reply src/models/logic/utils/workerpool.ts]: fixed

**Pull request merged**